ynr/apps/official_documents/templates/official_documents/officialdocument_detail.html

```diff
@@ -1,45 +0,0 @@
-{% extends "base.html" %}
-{% load i18n %}
-{% load pipeline %}
-
-
-{% block extra_css %}
-{% stylesheet 'official_documents' %}
-{% endblock %}
-
-{% block hero %}<h1>{{ object.document_type }}</h1>{% endblock hero %}
-
-{% block content %}
-
-<p>
-
-  {{ object.document_type }} for <a href="{{ object.post_election.get_absolute_url }}">{{ post_label }}</a>.
-
-{% blocktrans trimmed with source_url=object.source_url %}
-The source URL for this document was: <a href="{{ source_url }}">{{ source_url }}</a>
-{% endblocktrans %}
-</p>
-
-{% if documents_with_same_source.exists and documents_with_same_source.count > 1 %}
-<p>
-  <strong>
-  Watch out! This document contains candidate info for {{ documents_with_same_source.count }}
-  area{{ documents_with_same_source.count|pluralize }}</strong>.
-  Make sure you're looking at the right area.
-</p>
-{% endif %}
-
-<iframe
-  src="https://docs.google.com/viewer?url={% if object.uploaded_file.url|slice:":4" != "http" %}http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}{% endif %}{{ object.uploaded_file.url | urlencode}}&amp;embedded=true"
```

Comment:
One of the nice things about the google docs iframe is that it could deal with word docs. Switching to pdf.js breaks in this case, for example: https://candidates.democracyclub.org.uk/upload_document/10823/

How about either keeping google docs viewer as a fallback for non-PDF SOPNs?